client/profile-wizard/steps/industry.js

```diff
@@ -252,16 +254,21 @@ class Industry extends Component {
 
 export default compose(
 	withSelect( ( select ) => {
-		const { getProfileItems, getOnboardingError } = select(
-			ONBOARDING_STORE_NAME
-		);
+		const {
+			getProfileItems,
+			getOnboardingError,
+			isOnboardingRequesting,
+		} = select( ONBOARDING_STORE_NAME );
 		const { getSettings } = select( SETTINGS_STORE_NAME );
 		const { general: locationSettings = {} } = getSettings( 'general' );
 
 		return {
 			isError: Boolean( getOnboardingError( 'updateProfileItems' ) ),
 			profileItems: getProfileItems(),
 			locationSettings,
+			isProfileItemsRequesting: isOnboardingRequesting(
```

Comment:
Very nitpicky, but can we update all of these to `isUpdateRequesting` or `isUpdatingProfileItems`?  This can sometimes be ambiguous inside the render as to whether this is referring to the request to get items or to update them.

Reply:
@joshuatf Not at all :) Thank you for the suggestion. I think it's cleaner that way. Updated in https://github.com/woocommerce/woocommerce-admin/pull/5704/commits/e99d88f131cfcd8715de3e87936731877ee8940a

Reply:
For the [bbject Object] and error, I will run more tests and see what I can find.

Reply:
I tested it a few times with different internet speed (used throttling in chrome), but I could not reproduce it.